languages/go/types/polar_types.go

```diff
@@ -405,28 +385,21 @@ func (result *MessageKind) UnmarshalJSON(b []byte) error {
 		}
 		*result = MessageKind{variant}
 		return nil
-
 	}
 
-	return fmt.Errorf("Cannot deserialize MessageKind: %s", string(b))
+	return fmt.Errorf("cannot deserialize MessageKind: %s", string(b))
 }
 
 func (variant MessageKind) MarshalJSON() ([]byte, error) {
 	switch inner := variant.MessageKindVariant.(type) {
-
 	case MessageKindPrint:
-		return json.Marshal(map[string]MessageKindPrint{
-			"Print": inner,
-		})
-
+		return json.Marshal("Print")
```

Comment:
This is the main result of updating the codegen: we now serialize unit variants as the string, not with an empty inner thing.